ui/tests/acceptance/config-ui/login-settings-test.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click, visit, currentRouteName } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { runCmd } from 'vault/tests/helpers/commands';
+
+module('Acceptance | Enterprise | config-ui/login-settings', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    await login();
+
+    // create login rules
+    await runCmd([
+      `write sys/config/ui/login/default-auth/testRule backup_auth_types=userpass default_auth_type=okta disable_inheritance=false namespace=ns1`,
+      'write sys/config/ui/login/default-auth/testRule2 backup_auth_types=oidc default_auth_type=ldap disable_inheritance=true namespace=ns2',
+    ]);
+  });
+
+  hooks.afterEach(async function () {
+    // cleanup login rules
```

Comment:
So these commands run, I think `login()` needs to be called first.  To debug, you should be able to pass `true` as the second arg for [throwErrors](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/commands.js#L45) to confirm